recipes/mox/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: mox
+  version: {{ version }}
+
+source:
+  fn: mox-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/m/mox/mox-{{ version }}.tar.gz
+  md5: 6de7371e7e8bd9e2dad3fef2646f4a43
+
+build:
+  number: 0
+  skip: true  # [py3k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python <3
+
+  run:
+    - python <3
```

Comment:
See https://travis-ci.org/conda-forge/staged-recipes/jobs/117549022#L321 and you'll notice the lacking of py27 in the version string.